presenter/gitiles/gitiles_test.go

```diff
@@ -22,7 +23,7 @@ func TestFetchLog(t *testing.T) {
 		}),
 	}
 
-	log, err := fetchLog(client, "")
+	log, err := fetchLog(context.Background(), client, "")
```

Comment:
Thanks for your comment @slimsag!

`context.TODO()` exists so you can write code like this:

```Go
fetchLog(nil /* TODO: Figure out what context I want to use here. */, client, "")
```

See [its documentation](https://godoc.org/context#TODO):

> TODO returns a non-nil, empty Context. Code should use context.TODO when it's unclear which Context to use or it is not yet available (because the surrounding function has not yet been extended to accept a Context parameter). TODO is recognized by static analysis tools that determine whether Contexts are propagated correctly in a program.

And `context.Background()` doesn't mean mean it's something that should run in background, as far as I know. [Its documentation](https://godoc.org/context#Background) says:

> Background returns a non-nil, empty Context. It is never canceled, has no values, and has no deadline. It is typically used by the main function, initialization, and tests, and as the top-level Context for incoming requests.

Which is exactly what I want.

Basically, `context.TODO()` means the code is unfinished (and you're not sure which context you're going to use). `context.Background()` is just a normal top-level empty context.

Also see https://github.com/google/go-github/commit/ed4cdcede68dee8b12b2e4c1d7cdf0cb13ac851f which is relevant.